src/main.rs

```diff
@@ -38,9 +42,46 @@ mod ticker;
 fn main() {
     let args = parse_args();
 
-    simple_logger::init_with_level(args.log_level).unwrap();
+    let config = match args.log_config {
+        Some(path) => {
+            // Register deserializer for syslog so we can load syslog appender(s)
+            let mut deserializers = log4rs::file::Deserializers::new();
+            log4rs_syslog::register(&mut deserializers);
+
+            let mut config = match log4rs::load_config_file(path, deserializers) {
+                Ok(x) => x,
+                Err(_) => process::exit(1),
```

Comment:
Would `?` be sufficient, or are we going to consider these non-fatal errors?

Reply:
main doesn't return a result, so question mark seems out.

In the first case, it would seem like the file provided is either missing or inaccurate, in which case, you should default to console logging and log the error.  In the second case, it would look like the console logger is badly configured (programmer error?), so probably also safe to do an `eprintln!` and exit.